src/ui/public/utils/scanner.js

```diff
@@ -38,6 +38,11 @@ Scanner.prototype.scanAndMap = function (searchString, options, mapFn) {
 
   return new Promise((resolve, reject) => {
     const getMoreUntilDone = (error, response) => {
+      if (error) {
+        console.error(error);
+        resolve(allResults);
```

Comment:
I think it's important that we fail with an error message rather than bail out gracefully with only a subset of the results.  If we export only some of the results, people likely won't have any idea that an error occurred, and it could be very difficult to debug.  For example, if they were exporting 200 objects to import into another install, and only 120 objects actually exported, they probably won't notice that the other 80 weren't in there.